Modules/DRSRule.md

```diff
@@ -0,0 +1,17 @@
+# Name: DRSRule
+
+## Description:
+The DRSRule module allows you to work with all types of vSphere DRS rules. The module provides support for VM and VMHost groups. And, it works with affinity/anti-affinity VM rules and VM to VMHost rules.
+
+| Works with | Author(s)  | Project URL(s) | Last Update   | Tags |
+|------------|------------|----------------|---------------|------|
+| vSphere 5+ | Matt Boren | [GitHub](https://https://github.com/PowerCLIGoodies/DRSRule) | May 2016 | vSphere, Cluster, DRS, Rules, Groups |
```

Comment:
The URL has http:// twice
Please add both authors name on the same cell and if you don't mind if you want, you should add the b | t with links for each author